code/modules/antagonists/clock_cult/enchantments/blocking.dm

```diff
@@ -3,6 +3,5 @@
 
 /datum/component/enchantment/blocking/apply_effect(obj/item/target)
 	examine_description = "It has been blessed with the gift of blocking."
-	target.block_level = min(level - 1, 1)
+	target.canblock = TRUE
```

Comment:
flags probably. Maybe not actually since this would be cooler if they didn't require active hand.